dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/CodecSupport.java

```diff
@@ -41,6 +44,8 @@
     private static Map<Byte, Serialization> ID_SERIALIZATION_MAP = new HashMap<Byte, Serialization>();
     private static Map<Byte, String> ID_SERIALIZATIONNAME_MAP = new HashMap<Byte, String>();
     private static Map<String, Byte> SERIALIZATIONNAME_ID_MAP = new HashMap<String, Byte>();
+    // Cache null object serialize results, for heartbeat request/response serialize use.
+    private static Map<Byte, byte[]> ID_NULLBYTES_MAP = new HashMap<Byte, byte[]>();
```

Comment:
hi, @zhangyz-hd , will this map causes memory leak

Reply:
There will be no memory leaks,  it just cache the bytes of each Serialization-SPI null object when used. For example, '4e' for "hessian2", and '00' for 'kryo'.

Reply:
thanks for your explanation